packages/components/src/components/link-tile-model-signature/link-tile-model-signature-styles.ts

```diff
@@ -50,13 +50,15 @@ export const getComponentCss = (
     },
     heading: {
       margin: 0,
-      ...textLargeStyle,
+      font: textLargeStyle.font,
```

Comment:
I wouldn't only access `.font`, what happens if we extend the css rules for `textLargeStyles` in the future?

Reply:
And what happend to the previous auto hyphenation styles?

Reply:
Maybe `...textLargeStyle, hypens: 'inherit'` would be more future proof?